diff --git a/src/schema.rs b/src/schema.rs
index 8aa554f..d3fbf8e 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,13 +1,13 @@
 table! {
     use diesel::sql_types::Uuid;
-    use crate::db::models::PeriodicSubscriptionFrequency;
+    use crate::db::models::PeriodicSubscriptionFrequencyMapping;
     periodic_subscriptions (periodic_subscription_id) {
         periodic_subscription_id -> Uuid,
         subscription_id -> Uuid,
-        frequency -> PeriodicSubscriptionFrequency,
+        frequency -> PeriodicSubscriptionFrequencyMapping,
     }
 }
 
 table! {
     sent (sent_id) {
         sent_id -> Uuid,
@@ -16,17 +16,17 @@ table! {
         sent_date_time -> Timestamp,
     }
 }
 
 table! {
     use diesel::sql_types::Uuid;
-    use crate::db::models::MeansOfTransportation;
+    use crate::db::models::MeansOfTransportationMapping;
     sent_means (sent_means_id) {
         sent_means_id -> Uuid,
         user_id -> Uuid,
-        means -> MeansOfTransportation,
+        means -> MeansOfTransportationMapping,
     }
 }
 
 table! {
     sent_versions (sent_version_id) {
         sent_version_id -> Uuid,
@@ -36,18 +36,18 @@ table! {
     }
 }
 
 table! {
     use diesel::sql_types::Uuid;
     use diesel::sql_types::Text;
-    use crate::db::models::SubscriptionType;
+    use crate::db::models::SubscriptionTypeMapping;
     subscriptions (subscription_id) {
         subscription_id -> Uuid,
         user_id -> Uuid,
         crate_name -> Text,
-        subscription_type -> SubscriptionType,
+        subscription_type -> SubscriptionTypeMapping,
     }
 }
 
 table! {
     users (user_id) {
         user_id -> Uuid,
